typescript/src/processors/readTypeScriptModules/index.ts

```diff
@@ -151,13 +152,26 @@ export class ReadTypeScriptModules implements Processor {
     private addMemberDocs(docs: DocCollection, members: MemberDoc[]) {
       members.forEach(member => {
         docs.push(member);
-        if (member instanceof MethodMemberDoc) member.overloads.forEach(overloadDoc => docs.push(overloadDoc));
+        if (member instanceof MethodMemberDoc) {
+          this.addParamDocs(docs, member.parameterDocs);
+          member.overloads.forEach(overloadDoc => {
+            docs.push(overloadDoc);
```

Comment:
Won't that be "exposed" already? e.g. in line 136, where `functionDoc.overloads.forEach(doc => docs.push(doc))` is being called?

Reply:
This is for overloads of members of a exported class (or interface, etc), which are different to overloads of a freestanding exported function.

Reply:
Gotcha. Thanks for the explanation